### PR TITLE
[DOC] output description

### LINKED ISSUE #61: Describe output(s) in output.md
Example: https://github.com/nf-core/rnaseq/blob/master/docs/output.md
- [ ] logs
- [ ] proteomicslfq output
  - [ ] consensusXML
  - [ ] mzTab
  - [ ] msstats.tsv
- [ ] idXMLs
- [ ] msstats output
  - [ ] table of quants (tsv)
  - [ ] VolcanoPlots
  - [ ] Heatmaps
  - [ ] ComparisonPlots
- [ ] ptxqc report

### PR DESCRIPTION
**Pull request by jpfeuffer**
As release requirement.

fixes #61 

TODO:
 - [ ] address TODOs
 - [ ] add pictures
 - [ ] add links to references

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -1,10 +1,105 @@
...
+  * [ComparisonPlot.pdf](#msstats-plots)
+  * [VolcanoPlot.pdf](#msstats-plots)
+  * [Heatmap.pdf](#msstats-plots)
+  * [msstats\_results.csv](#msstats-table)
+* pipeline\_info
+  * [...](#nextflow-pipeline-info)
+* proteomics\_lfq
+  * [debug\_\*.idXML](#debug-output)
+  * [out.consensusXML](#consenusxml)
+  * [out.csv](#msstats-ready-quantity-table)
```
Comment (ypriverol) [inline docs/output.md:41]: This table is protein or peptide tables?

Comment (jpfeuffer) [reply docs/output.md:41]: Protein level

File: docs/output.md [hunk 2]
```diff
@@ -1,10 +1,115 @@
...
+### Nextflow pipeline info
+
+Information about the execution and structure of the pipeline. If run with the corresponding nextflow parameters,
+it can include things like a visual representation of the pipeline and/or a html report on the execution with
+info on memory consumption, CPU usage and runtimes.
+
+### Identifications
+
+Intermediate output for the PSM/peptide-level filtered identifications per raw/mzML file in OpenMS'
+internal [idXML](https://github.com/OpenMS/OpenMS/blob/develop/share/OpenMS/SCHEMAS/IdXML_1_5.xsd) format. TODO link to schema.
```
Comment (timosachsenberg) [inline docs/output.md:64]: TODO?

File: docs/output.md [hunk 3]
```diff
@@ -1,10 +1,115 @@
...
+
+See [MSstats vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/MSstats/inst/doc/MSstats.html).
+
+#### MSstats plots
+
+See [MSstats vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/MSstats/inst/doc/MSstats.html) for groupComparisonPlots (Heatmap, VolcanoPlot and ComparisonPlot (per protein)).
+
+### PTXQC output
+
+If activated, the `ptxqc` folder will contain the report of the [PTXQC R package](https://cran.r-project.org/web/packages/PTXQC/index.html) based on the mzTab output of proteomicsLFQ.
```
Comment (timosachsenberg) [inline docs/output.md:106]: does this already work?

**Pull request merged**